### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**
image
image
image
image
image
image
image
image

### CODE REVIEW

**Review by matechris — APPROVED** [review]
Looks good :)

File: templates/taxi/car_detail.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <p><strong>Manufacturer:</strong> ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
```
Comment (Y-Havryliv) [inline templates/taxi/car_detail.html:4]: maybe add car model?

File: templates/taxi/car_list.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Car List</h1>
```
Comment (Y-Havryliv) [inline templates/taxi/car_list.html:4]: Create table with the information of each instance

File: templates/taxi/car_list.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  {% if car_list %}
+    <ul>
+      {% for car in car_list %}
+        <li>
+          <a href="{% url "taxi:car-detail" pk=car.id %}">{{ car.id }} {{ car.model }}</a>
```
Comment (Y-Havryliv) [inline templates/taxi/car_list.html:9]: In the car list set a link on the id field that leads to the car detail page.

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

File: templates/taxi/car_list.html (hunk 3 above)
Comment (HStakhiv) [reply templates/taxi/car_list.html]: Done
image

File: templates/taxi/car_list.html (hunk 2 above)
Comment (HStakhiv) [reply templates/taxi/car_list.html]: Done
image

File: templates/taxi/car_list.html (hunk 3 above)
Comment (HStakhiv) [reply templates/taxi/car_list.html]: image

**Review by Y-Havryliv — APPROVED** [review]
Well done!

**Pull request closed without merging**